components/Player/PlayerShareMenu/PlayerShareMenu.tsx

```diff
@@ -55,16 +57,20 @@ const PlayerShareMenu: React.FC<IPlayerShareMenuProps> = ({
 
           <CopyLinkButton />
 
-          <FileDownloadButton />
+          {!isPrivate && (
```

Comment:
I guess I would have preferred using the existing pattern - allowing the caller to override the download-button-url via a query parameter.  (Or passing an empty/blank value to indicate the button shouldn't be shown at all).  Would have been a bit more flexible.

But, many ways to do the same thing, and this gets the job done.

Reply:
That var is from the component props, but the value passed to the props coms from the embed config, which is parsed from the URL params. However, I didn't map the config object prop to a URL param key. We can easily add a param to the IEmbedConfig maps.